pkg/volume/fc/fc_util.go

```diff
@@ -196,6 +196,10 @@ func (util *fcUtil) MakeGlobalVDPDName(fc fcDisk) string {
 	return makeVDPDNameInternal(fc.plugin.host, fc.wwns, fc.lun, fc.wwids)
 }
 
+func parseDeviceName(devicePath string) string {
+	return strings.TrimPrefix(devicePath, "/dev/")
+}
```

Comment:
This is used only in unit tests, please move it to fc_util_test.go. If it is needed at all.